sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusMessageSerializer.java

```diff
@@ -225,6 +231,37 @@ public int getSize(Message amqpMessage) {
         return amqpMessage;
     }
 
+    private Map<String, Object> convertToDescribedType(Map<String, Object> propertiesValue) {
+        for (Map.Entry<String, Object> entry : propertiesValue.entrySet()) {
+            Object value = entry.getValue();
+            if (value instanceof URI) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.URI_SYMBOL, ((URI) value).toString()));
+            } else if (value instanceof OffsetDateTime) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.OFFSETDATETIME_SYMBOL, ((OffsetDateTime) value).atZoneSameInstant(ZoneOffset.UTC).toInstant().toEpochMilli()));
+            } else if (value instanceof Duration) {
+                propertiesValue.put(entry.getKey(), new ServiceBusDescribedType(ServiceBusConstants.DURATION_SYMBOL, ((Duration) value).toNanos()));
```

Comment:
Could you double check that it is compatible with .NET?

https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.utcticks?view=net-6.0
https://docs.microsoft.com/en-us/dotnet/api/system.timespan.ticks?view=net-6.0

Both seems in 100-nano or 1/10-milli.

Reply:
I see. I'll try to fix here.

Reply:
Also check with Connie whether this would loop in Josh for cross-SDK, as we might want to check whether other language is on same format (if they do not have it, fine, but if they have it but using different format e.g. milli, they or we have big problem...).

Reply:
Yes, this may cause a accuracy issue. I know that golang support nanosecond. If all the SDK need to align with .net SDK, most of SDK need to trim the value in particular. And if we change the format in .net SDK, it will be a breaking change for .net SDK.